src/test/java/edu/harvard/iq/dataverse/branding/BrandingUtilTest.java

```diff
@@ -1,32 +1,46 @@
 package edu.harvard.iq.dataverse.branding;
 
+import edu.harvard.iq.dataverse.DataverseServiceBean;
 import edu.harvard.iq.dataverse.util.BundleUtil;
 import java.io.UnsupportedEncodingException;
 import java.util.Arrays;
 import javax.mail.internet.AddressException;
 import javax.mail.internet.InternetAddress;
 import static org.junit.Assert.assertEquals;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.junit.jupiter.MockitoExtension;
 
+@ExtendWith(MockitoExtension.class)
 public class BrandingUtilTest {
 
+    @Mock
+    DataverseServiceBean dataverseService;
+    
     @Test
     public void testGetInstallationBrandName() {
-        System.out.println("testGetInstallationBrandName");
-        assertEquals("LibraScholar", BrandingUtil.getInstallationBrandName("LibraScholar"));
-        assertEquals(null, BrandingUtil.getInstallationBrandName(null));// misconfiguration to set to null
-        assertEquals("", BrandingUtil.getInstallationBrandName(""));// misconfiguration to set to empty string
+        
+       System.out.println("testGetInstallationBrandName");
+        assertEquals("LibraScholar", BrandingUtil.getInstallationBrandName());
```

Comment:
what is the point of this test?

Reply:
It checks that the mock dataverseService is being seen within BrandingUtil and responding with 'LibraScholar'. FWIW: I think it works because this test runs after the next one which has line 38 to set the mock correctly. I can look at adding a redundant call here to make sure it works/be clearer. (Also note that there will be more subtests here when there's a installation.name setting that could change the output of this method.)

Reply:
yeah, these tests like these feel basically useless to me; i.e. it's effectively testing the mock service, rather than using the mock service to test something real. I'd vote to remove it altogether.

Reply:
removed